spec/requests/games_spec.rb

```diff
@@ -1,91 +1,105 @@
+require "rails_helper"
+
+RSpec.describe "Games Controller", type: :request do
+  context "create game" do
+    it "creates a game" do
+      headers = { "ACCEPT" => "application/json" }
+      post "/games", params: { game: { opponent: "computer", current_player: "human" } }, headers: headers
+      expect(response.content_type).to eq("application/json; charset=utf-8")
       expect(response).to have_http_status(:created)
     end
-    it 'fails to creates a game' do
-      headers = { 'ACCEPT' => 'application/json' }
-      post '/games', params: { game: { oppogdgdnent: 'computer' } }, headers: headers
-      expect(response.content_type).to eq('application/json; charset=utf-8')
-      expect(response.body).to include('error')
+    it "fails to creates a game" do
+      headers = { "ACCEPT" => "application/json" }
+      post "/games", params: { game: { oppogdgdnent: "computer" } }, headers: headers
+      expect(response.content_type).to eq("application/json; charset=utf-8")
+      expect(response.body).to include("error")
     end
   end
 
-  context 'play_game' do
-    it 'should move to position 2 on the game board' do
-      game = Game.create(opponent: 'human', current_player: 'player1', board: Array.new(9, '-'))
+  context "play_game" do
+    it "should move to position 2 on the game board" do
+      game = Game.create(opponent: "human", current_player: "player1", board: Array.new(9, "-"))
 
-      expected_result = { 'message'=>{'board'=>['-', '-', 'x', '-', '-', '-', '-', '-', '-']}}
+      expected_result = { "message" => { "board" => ["-", "-", "x", "-", "-", "-", "-", "-", "-"] } }
       patch "/games/#{game.id}/move/2"
 
-      expect(response.content_type).to eq('application/json; charset=utf-8')
+      expect(response.content_type).to eq("application/json; charset=utf-8")
       expect(response).to have_http_status(200)
 
       expect(JSON.parse(response.body)).to eq(expected_result)
     end
-    it 'should output player1 as the winner' do
-      game = Game.create(opponent: 'human', player: 'x', current_player: 'player1', board: ['-', '-', 'x', '-', '-', 'x', 'o', '-', 'x'])
+    it "should output player1 as the winner" do
+      game = Game.create(opponent: "human", player: "x", current_player: "player1", board: ["-", "-", "x", "-", "-", "x", "o", "-", "x"])
 
-      expected_result = {"board"=>["x", "-", "x", "-", "-", "x", "o", "-", "x"], "win"=>"player1 won the game"}
+      expected_result = { "board" => ["x", "-", "x", "-", "-", "x", "o", "-", "x"], "win" => "player1 won the game" }
       patch "/games/#{game.id}/move/0"
 
-      expect(response.content_type).to eq('application/json; charset=utf-8')
+      expect(response.content_type).to eq("application/json; charset=utf-8")
       expect(response).to have_http_status(200)
-      expect(JSON.parse(response.body)['message']).to eq(expected_result)
+      expect(JSON.parse(response.body)["message"]).to eq(expected_result)
```

Comment:
It looks like we reuse this code `JSON.parse(response.body)` in many places. Could we remove the duplication?